docs/guidance/map.md

```diff
@@ -25,8 +25,6 @@ Mapping data to OCDS is not always easy. Before writing any software, this phase
 
 The documentation you produce can also later be included in your [Data User Guide](publish.md#finalize-your-publication-policy).
 
-As you make progress through this phase, we encourage you to update your [publication plan](design.md#develop-your-publication-plan), in order to help set priorities and ease communication within your team, with your stakeholders, and with the OCDS Helpdesk. You can start by filling in the _Goals (design)_ section.
```

Comment:
What do you think of keeping the idea of the paragraph, like:

```md
As you make progress through this phase, we encourage you to maintain a project plan, in order to help set priorities and ease communication within your team, with your stakeholders, and with the OCDS Helpdesk.
```

Since it's true that having some shared document is a good idea - we just don't provide the template for it.